src/java/org/apache/cassandra/repair/autorepair/AutoRepairUtils.java

```diff
@@ -798,7 +803,7 @@ public static boolean checkNodeContainsKeyspaceReplica(Keyspace ks)
                 ksReplicaOnNode = false;
             }
         }
-        if (replicationStrategy instanceof LocalStrategy)
+        if (replicationStrategy instanceof LocalStrategy || replicationStrategy instanceof MetaStrategy)
```

Comment:
Need to tackle this separately as this is unique to trunk/5.1 https://issues.apache.org/jira/browse/CASSANDRA-20138